Generate.py

```diff
@@ -210,7 +210,7 @@ def main(args=None, callback=ERmain):
         else:
             raise RuntimeError(f'No weights specified for player {player}')
 
-    if len(set(name.lower() for name in erargs.name.values())) != len(erargs.name):
+    if len(set(name.lower().replace("_", " ") for name in erargs.name.values())) != len(erargs.name):
```

Comment:
I'm wondering if it would be cheaper to do `" " -> "_"` rather than `"_" -> " "` because during runtime we will see more `"_"` than `" "`, but it shouldn't matter much.